### ISSUE TITLE
Cloning of a multi-language item

### ISSUE THREAD

**Issue by MarcinOkon** [sergeyshushlyapin/Sitecore.FakeDb#120]
Hi
When trying to clone an item with multiple languages the non-english languages have no SourceUri set.

```c#
[Test]
public void TestCloning()
{
	//Arrange
	using (var db = new Db
	{
		new DbItem("SourceItem")
		{
			new DbField("Title")
			{
				{ "en", "Hello!" },
				{ "de", "Servus!" }
			}
		}
	})
	{
		Item source = db.GetItem("/sitecore/content/SourceItem");

		//Act
		source.CloneTo(source.Parent, "CloneItem", false);

		//Assert

		Item sourceEn = db.GetItem("/sitecore/content/SourceItem", "en");
		Item sourceDe = db.GetItem("/sitecore/content/SourceItem", "de");
		Item cloneEn = db.GetItem("/sitecore/content/CloneItem", "en");
		Item cloneDe = db.GetItem("/sitecore/content/CloneItem", "de");

		Assert.AreEqual(1, cloneEn.Versions.Count);
		Assert.AreEqual(1, cloneDe.Versions.Count);

		Assert.AreEqual(sourceEn.Uri, cloneEn.SourceUri);
		Assert.AreEqual(sourceDe.Uri, cloneDe.SourceUri); //fails here with null
	}
}
```

I have compared the CloneTo execution with a real sitecore instance and I think there is a difference in how both are handling field changes on items without a version.
Here is a test case that demonstrates it:

```c#
[Test]
public void TestEdit()
{
	using (var db = new Db
	{
		new DbItem("home")
		{
			new DbField("Title")
			{
				{"en", "Hello!"}
			}
		}
	})
	{

		Item homeEn = db.GetItem("/sitecore/content/home", "en");
		Item homeDe = db.GetItem("/sitecore/content/home", "de");

		Assert.AreEqual(1, homeEn.Versions.Count);
		Assert.AreEqual(0, homeDe.Versions.Count);
		
		using (new EditContext(homeEn))
		{
			homeEn.Fields["Title"].SetValue("Bye!", true);
		}

		using (new EditContext(homeDe))
		{
			homeDe.Fields["Title"].SetValue("Servus!", true);
		}
		
		Assert.AreEqual(1, homeDe.Versions.Count);
		Assert.AreEqual(1, homeEn.Versions.Count);

		Assert.AreEqual("Bye!", homeEn["Title"]);
		Assert.AreEqual("Servus!", homeDe["Title"]); //fails here with string.empty
	}
}
```

**Comment by sergeyshushlyapin** [comment]
@MarcinOkon, good catch, thanks! Fixed.

**Comment by sergeyshushlyapin** [comment]
Available as [v0.36.2](https://www.nuget.org/packages/Sitecore.FakeDb/).

**Comment by MarcinOkon** [comment]
thank you for the fast fix!
